.github/workflows/perl-package-build.yml

```diff
@@ -22,85 +22,84 @@ jobs:
+              ;; \
+            'mips64el') \
+              url='https://dl.google.com/go/go1.21.3.linux-mips64le.tar.gz'; \
+              sha256='a569ffbc88b4e14cf2682f65cec950460665e4392b0d78b8868b4718c979bda8'; \
+              ;; \
+            'ppc64el') \
+              url='https://dl.google.com/go/go1.21.3.linux-ppc64le.tar.gz'; \
+              sha256='3b0e10a3704f164a6e85e0377728ec5fd21524fabe4c925610e34076586d5826'; \
+              ;; \
+            'riscv64') \
+              url='https://dl.google.com/go/go1.21.3.linux-riscv64.tar.gz'; \
+              sha256='67d14d3e513e505d1ec3ea34b55641c6c29556603c7899af94045c170c1c0f94'; \
+              ;; \
+            's390x') \
+              url='https://dl.google.com/go/go1.21.3.linux-s390x.tar.gz'; \
+              sha256='4c78e2e6f4c684a3d5a9bdc97202729053f44eb7be188206f0627ef3e18716b6'; \
+              ;; \
+            *) echo >&2 "error: unsupported architecture '$arch' (likely packaging update needed)"; exit 1 ;; \
+          esac; \
+          build=; \
+          if [ -z "$url" ]; then \
+          # https://github.com/golang/go/issues/38536#issuecomment-616897960
+            build=1; \
+            url='https://dl.google.com/go/go1.21.3.src.tar.gz'; \
+            sha256='186f2b6f8c8b704e696821b09ab2041a5c1ee13dcbc3156a13adcf75931ee488'; \
+            echo >&2; \
+            echo >&2 "warning: current architecture ($arch) does not have a compatible Go binary release; will be building from source"; \
+            echo >&2; \
+          fi; \
+          \
+          wget -O go.tgz.asc "$url.asc"; \
+          wget -O go.tgz "$url" --progress=dot:giga; \
+          echo "$sha256 *go.tgz" | sha256sum -c -; \
+          \
+          # https://github.com/golang/go/issues/14739#issuecomment-324767697
+          GNUPGHOME="$(mktemp -d)"; export GNUPGHOME; \
+          # https://www.google.com/linuxrepositories/
+          gpg --batch --keyserver keyserver.ubuntu.com --recv-keys 'EB4C 1BFD 4F04 2F6D DDCC  EC91 7721 F63B D38B 4796'; \
+          # let's also fetch the specific subkey of that key explicitly that we expect "go.tgz.asc" to be signed by, just to make sure we definitely have it
+          gpg --batch --keyserver keyserver.ubuntu.com --recv-keys '2F52 8D36 D67B 69ED F998  D857 78BD 6547 3CB3 BD13'; \
+          gpg --batch --verify go.tgz.asc go.tgz; \
+          gpgconf --kill all; \
+          rm -rf "$GNUPGHOME" go.tgz.asc; \
+          \
+          tar -C /usr/local -xzf go.tgz; \
+          rm go.tgz;
+          /usr/local/go/bin/go version
+          echo "/usr/local/go/bin" >> $GITHUB_PATH
+          echo "${HOME}/go/bin" >> $GITHUB_PATH
 
       - name: Install one-off Bystro dependencies
         shell: bash
         run: |
           go install github.com/bystrogenomics/bystro-vcf@2.2.2
           cpm install -g https://github.com/bystrogenomics/msgpack-perl.git
           cpm install -g --no-test MouseX::Getopt
-          cpm install -g DBD::mysql@4.051
```

Comment:
no longer needed; mariadb has a smoother install process (and is compatible with mysql; tested the only mysql use we have in Perl, namely bystro-utils.pl fetching utility)